app/views/admin/amr_data_feed_configs/edit.html.erb

```diff
@@ -2,6 +2,7 @@
 
 <%= simple_form_for [:admin, @configuration] do |f| %>
   <%= f.input :description, as: :string, hint: 'A human-readable name for the configuration' %>
+  <%= f.input :owned_by_id, collection: User.admin, hint: 'Admin contact for the configuration' %>
```

Comment:
Sort by name, and use the name as the label? Also allow blank so a user can be removed?